### ISSUE TITLE
power down the device to fix iOS white screen?

### ISSUE THREAD

**Issue by remakerich** [jonbhanson/flutter_native_splash#342]
**Describe the bug**
There's white splash screen on iOS when launching the app before desired splash appears.

**To Reproduce**
1. flutter build ipa
2. *open archive in xcode*
3. distribute app
4. Development
5. upload .ipa to Firebase app Distribution

**Expected behavior**
Splash screen starts right from the app launch

**Flutter doctor**
[✓] Flutter (Channel stable, 2.10.4, on
    macOS 12.3.1 21E258 darwin-arm,
    locale ru-RU)
    • Flutter version 2.10.4 at
      /Users/kpopovskikh/flutter
    • Upstream repository
      https://github.com/flutter/flutte
      r.git
    • Framework revision c860cba910 (5
      weeks ago), 2022-03-25 00:23:12
      -0500
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[✓] Android toolchain - develop for
    Android devices (Android SDK
    version 32.1.0-rc1)
    • Android SDK at
      /Users/kpopovskikh/Library/Androi
      d/sdk
    • Platform android-32, build-tools
      32.1.0-rc1
    • Java binary at:
      /Applications/Android
      Studio.app/Contents/jre/Contents/
      Home/bin/java
    • Java version OpenJDK Runtime
      Environment (build
      11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    (Xcode 13.2.1)
    • Xcode at
      /Applications/Xcode.app/Contents/
      Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google
      Chrome.app/Contents/MacOS/Google
      Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at
      /Applications/Android
      Studio.app/Contents
    • Flutter plugin can be installed
      from:
      🔨
      https://plugins.jetbrains.com/plu
      gin/9212-flutter
    • Dart plugin can be installed
      from:
      🔨
      https://plugins.jetbrains.com/plu
      gin/6351-dart
    • Java version OpenJDK Runtime
      Environment (build
      11.0.11+0-b60-7772763)

[✓] VS Code (version 1.66.2)
    • VS Code at /Applications/Visual
      Studio Code.app/Contents
    • Flutter extension version 3.38.1

[✓] Connected device (3 available)
    • iPhone - Поповских К (mobile) •
      521a926799df5c6cf6a865d69f900f086
      3f9b14c • ios            • iOS
      15.1 19B74
    • iPhone 13 (mobile)            •
      2E8D686F-6003-4865-8239-B741D342C
      D8E     • ios            •
      com.apple.CoreSimulator.SimRuntim
      e.iOS-15-2 (simulator)
    • Chrome (web)                  •
      chrome
      • web-javascript • Google Chrome
      100.0.4896.127

[✓] HTTP Host Availability
    • All required HTTP hosts are
      available

• No issues found!

**Additional context**
I know there's a note in the readme:

"If the splash screen was not updated correctly on iOS or if you experience a white screen before the splash screen, run flutter clean and recompile your app. If that does not solve the problem, delete your app, power down the device, power up the device, install and launch the app as per [this StackOverflow thread](https://stackoverflow.com/questions/33002829/ios-keeping-old-launch-screen-and-app-icon-after-update)."

I have cleaned my build several times and recompiled and uploaded it again but white screen still persists.

I managed to fix that only after powering down the device.

It seems to me a complete madness to ask all your customers to power down the device.

Is the problem in native_splash_screen package?

**Comment by jonbhanson** [comment]
> Is the problem in native_splash_screen package?

No, the problem is in iOS.  Note that the [thread](https://stackoverflow.com/questions/33002829/ios-keeping-old-launch-screen-and-app-icon-after-update) referenced this problem in 2015, four years before this package was first developed.

I agree that it is complete madness that this problem remains after 7 years.

Note that your customers will only be affected if they currently have your app installed with a different splash screen.  For new installs, this problem does not occur.
